### ISSUE TITLE
Typescript server error : RangeError: out of range index

### ISSUE THREAD

**Issue by abdulkareemnalband** [Microsoft/vscode#34276]
- VSCode Version: Code 1.16.0 (787b31c0474e6165390b5a5989c9619e3e16f953, 2017-09-06T16:27:49.891Z)
- OS Version: Windows_NT x64 10.0.15063
- Typescript:2.4.2
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|0.16.0
iis-express-executer|And|1.2.5
ng-template|Ang|0.1.7
scratchpad|awe|0.1.0
htmltagwrap|bra|0.0.3
path-intellisense|chr|1.4.2
bracket-pair-colorizer|Coe|0.10.9
python|don|0.7.0
xml|Dot|1.9.2
EditorConfig|Edi|0.10.1
tslint|eg2|1.0.3
vscode-npm-script|eg2|0.2.1
vscode-npm|fkn|3.3.0
json2ts|Gre|0.0.6
docthis|joe|0.4.8
angular-essentials|joh|0.2.0
Angular2|joh|2.7.0
codeshell|Kip|0.2.2
typelens|kis|1.4.1
csharpfixformat|Leo|0.0.34
MagicPython|mag|1.0.12
csharp|ms-|1.12.1
mono-debug|ms-|0.15.7
team|ms-|1.122.0
debugger-for-chrome|msj|3.2.1
angular2-inline|nat|0.0.17
vscode-code-outline|pat|0.0.10
material-icon-theme|PKi|2.2.1
typescript-hero|rbb|1.4.1
vscode-icons|rob|7.12.0
vscode-sass-format|sas|1.1.1
html5-boilerplate|sid|1.0.1
python|tht|0.2.3
iis-express|war|1.0.0

---

Steps to Reproduce:

1.Open any typescript file
2.`Ctrl+click` on any symbol
3. See loader bar on top symbol is never resolved

Reproduces without extensions: Yes
logs
```
[Info  - 13:32:49] Using tsserver from: d:\VSTS\Decisions\Features\Figg.Decisions.Web\node_modules\typescript\lib
[Info  - 13:32:49] TSServer log file: C:\Users\ABDULK~1\AppData\Local\Temp\vscode-tsserver-log-cWlYzs\tsserver.log
[Info  - 13:32:49] Forking TSServer
PATH: C:\Python27\;C:\Python27\Scripts;C:\Program Files\Microsoft MPI\Bin\;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Program Files\Python36\Scripts\;C:\Program Files\Python36\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\nodejs\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\dotnet\;C:\ProgramData\chocolatey\bin;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\Calibre2\;C:\Program Files\Git\cmd;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Xoreax\IncrediBuild;D:\TEE-CLC-14.118.0;C:\Program Files (x86)\Yarn\bin;C:\Program Files (x86)\Graphviz2.38\bin;D:\utils\gettext-tools\bin;C:\Ruby24-x64\bin;C:\Users\AbdulkareemNalband\.cargo\bin;C:\Users\AbdulkareemNalband\AppData\Roaming\npm;C:\Users\AbdulkareemNalband\AppData\Local\Microsoft\WindowsApps;C:\Users\AbdulkareemNalband\AppData\Local\Yarn\bin;C:\Program Files\Bandizip\;C:\Program Files\Microsoft VS Code\bin;C:\Users\AbdulkareemNalband\AppData\Local\Programs\Fiddler
[Error  - 13:32:59] ReaderError
RangeError: out of range index
	at ProtocolBuffer.tryReadContent (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:74:21)
	at Reader.onLengthData (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:100:41)
	at Socket.Reader.readable.on (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:87:18)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at Pipe.onread (net.js:560:20)
[Error  - 13:32:59] ReaderError
RangeError: out of range index
	at ProtocolBuffer.tryReadContent (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:74:21)
	at Reader.onLengthData (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:100:41)
	at Socket.Reader.readable.on (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:87:18)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at Pipe.onread (net.js:560:20)
[Error  - 13:32:59] ReaderError
RangeError: out of range index
	at ProtocolBuffer.tryReadContent (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:74:21)
	at Reader.onLengthData (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:100:41)
	at Socket.Reader.readable.on (c:\Program Files\Microsoft VS Code\resources\app\extensions\typescript\out\utils\wireProtocol.js:87:18)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at Pipe.onread (net.js:560:20)

```

**Comment by abdulkareemnalband** [comment]
On changing typescript from workspace version 2.4.2 to vscode version 2.5.2 everything works fine

**Comment by mjbvz** [comment]
Ok, sounds like this was fixed by TS 2.5.2 them. Closing as upstream
